src/mock-doc/test/event.spec.ts

```diff
@@ -90,6 +90,48 @@ describe('event', () => {
     expect(ev.detail).toBe(88);
   });
 
+  it('FocusEvent() requires type', () => {
+    expect(() => {
+      // @ts-ignore checking that it throws when not supplied required arguments
+      new win.FocusEvent();
+    }).toThrow();
+  });
+
+  it('FocusEvent(type)', () => {
```

Comment:
just for completion's sake, what would you think about parameterizing this test across `"blur", "focus"`? something like this perhaps:

```suggestion
  it.each(["blur", "focus"])('creates a %s-type MockFocusEvent', (focusType) => {
```